[2025-07-09T05:03:22.524+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-09T05:03:22.700+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bash_call_python_file.extract_task manual__2025-07-09T05:03:18.296639+00:00 [queued]>
[2025-07-09T05:03:22.733+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bash_call_python_file.extract_task manual__2025-07-09T05:03:18.296639+00:00 [queued]>
[2025-07-09T05:03:22.734+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-09T05:03:22.797+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_task> on 2025-07-09 05:03:18.296639+00:00
[2025-07-09T05:03:22.902+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=202) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-09T05:03:22.881+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'bash_call_python_file', 'extract_task', 'manual__2025-07-09T05:03:18.296639+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpnj9gsyyo']
[2025-07-09T05:03:22.905+0000] {standard_task_runner.py:91} INFO - Job 31: Subtask extract_task
[2025-07-09T05:03:22.905+0000] {standard_task_runner.py:63} INFO - Started process 203 to run task
[2025-07-09T05:03:23.300+0000] {task_command.py:426} INFO - Running <TaskInstance: bash_call_python_file.extract_task manual__2025-07-09T05:03:18.296639+00:00 [running]> on host fe3fd911cec1
[2025-07-09T05:03:24.163+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='bash_call_python_file' AIRFLOW_CTX_TASK_ID='extract_task' AIRFLOW_CTX_EXECUTION_DATE='2025-07-09T05:03:18.296639+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-09T05:03:18.296639+00:00'
[2025-07-09T05:03:24.169+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-09T05:03:25.194+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-09T05:03:25.195+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-09T05:03:25.236+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bash_call_python_file, task_id=extract_task, run_id=manual__2025-07-09T05:03:18.296639+00:00, execution_date=20250709T050318, start_date=20250709T050322, end_date=20250709T050325
[2025-07-09T05:03:25.284+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-09T05:03:25.366+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-09T05:03:25.371+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
